### ISSUE TITLE
Reminders

### ISSUE THREAD

**Issue by alemaoJulian** [nvbn/everpad#325]
Reminders would be a nice additions!

**Comment by e2jk** [comment]
Agreed. And the ability to display e.g. reminders for today or those in the past that haven't been marked as done yet.

**Comment by thotmx** [comment]
+1 it could be nice.

**Comment by ameersohail0** [comment]
reminders would be the coolest thing to come in everpad.
